packages/dev/gui/src/2D/controls/container.ts

```diff
@@ -38,6 +38,12 @@ export class Container extends Control {
     /** @internal */
     protected _intermediateTexture: Nullable<DynamicTexture> = null;
 
+    /**
+     * Gets or sets a boolean indicating that the button will let internal controls handle picking instead of doing it directly using its bounding info

```

Comment:
```suggestion
     * Gets or sets a boolean indicating that the container will let internal controls handle picking instead of doing it directly using its bounding info
```